deprecated.json

```diff
@@ -0,0 +1,7 @@
+{
+  "components": [
+    { "componentName": "IconAsButton", "className": "sg-icon-as-button" },
+    { "componentName": "ContentBox", "className": "sg-content-box" },
+    { "componentName": "ActionList", "className": "sg-actions-list" }
+  ]
```

Comment:
PopupMenu & MenuList should be added here as well

Reply:
@katarzynatobis thanks! I will add

Reply:
done